File: MultiLanguageSandbox/src/test/deps/julia/44.jl
function compare_area(a::Int, b::Int, c::Int) :: String
    """
    Determine whose geometric shape has a greater area, Alice with her square of side length `a`, 
    or Bob with his rectangle of dimensions `b` x `c`. Returns "Alice" if the square's area is larger, 
    and "Bob" if the rectangle's area is larger.

    # Examples
    - `compare_area(5, 4, 6)` returns "Alice" because the area of the square (25) is greater than the area of the rectangle (24).
    - `compare_area(7, 5, 10)` returns "Bob" because the area of the rectangle (50) is greater than the area of the square (49).
    """
    square_area = a * a
    rectangle_area = b * c
    
    if square_area > rectangle_area
        return "Alice"
    else
        return "Bob"
    end
end
@assert compare_area(5, 4, 6) == "Alice"
@assert compare_area(7, 5, 10) == "Bob"
@assert compare_area(2, 3, 1) == "Alice"  # Testing with a smaller square and rectangle, square's area is 4, rectangle's area is 3
@assert compare_area(8, 4, 4) == "Alice"

File: MultiLanguageSandbox/src/test/deps/C#/44.cs
using System;
using System;
using System.Collections.Generic;
using System.Diagnostics;

class Program
{

/* Determines whose geometric shape has a larger area: Alice's square with side length a, or Bob's rectangle with sides b and c.
    - If Alice's square has a larger area, return "Alice".
    - If Bob's rectangle has a larger area, return "Bob".
    
    Examples:
    >>> CompareArea(5, 4, 6)
    "Alice"
    
    >>> CompareArea(7, 5, 10)
    "Bob"
*/

static string CompareArea(int a, int b, int c)
{
        // Calculate the area of Alice's square
        int aliceArea = a * a;
        
        // Calculate the area of Bob's rectangle
        int bobArea = b * c;
        
        // Compare the areas and return the appropriate result
        if (aliceArea > bobArea)
        {
            return "Alice";
        }
        else
        {
            return "Bob";
        }
    }

    // Example usage
    static void Main()
    {
        Debug.Assert(CompareArea(5, 4, 6) == "Alice");
        Debug.Assert(CompareArea(7, 5, 10) == "Bob");
        Debug.Assert(CompareArea(2, 2, 8) == "Bob"); // Testing with a square of side 2 and a rectangle 2x8
        Debug.Assert(CompareArea(10, 5, 5) == "Alice"); // Testing with a square of side 10 and a rectangle 5x5


    }
}

